contracts/zero-ex/contracts/test/foundry/WethTransformerTests.sol

```diff
@@ -0,0 +1,114 @@
+pragma solidity ^0.6;
+
+pragma experimental ABIEncoderV2;
+
+import "./utils/ForkUtils.sol";
+import "./utils/TestUtils.sol";
+import "src/IZeroEx.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+import "src/features/TransformERC20Feature.sol";
+import "src/external/TransformerDeployer.sol";
+import "src/transformers/WethTransformer.sol";
+import "src/transformers/FillQuoteTransformer.sol";
+import "src/transformers/bridges/BridgeProtocols.sol";
+import "src/transformers/bridges/EthereumBridgeAdapter.sol";
+import "src/transformers/bridges/PolygonBridgeAdapter.sol";
+import "src/transformers/bridges/ArbitrumBridgeAdapter.sol";
+import "src/transformers/bridges/OptimismBridgeAdapter.sol";
+import "src/transformers/bridges/AvalancheBridgeAdapter.sol";
+import "src/transformers/bridges/FantomBridgeAdapter.sol";
+import "src/transformers/bridges/CeloBridgeAdapter.sol";
+import "src/features/OtcOrdersFeature.sol";
+
+contract transformERC20Tests is Test, ForkUtils, TestUtils {
+    //use forge-std json library for strings
+    using stdJson for string;
+
+    //utility mapping to get chainId by name
+    mapping(string => string) public chainsByChainId;
+    //utility mapping to get indexingChainId by Chain
+    mapping(string => string) public indexChainsByChain;
+
+    string json;
+
+    function setUp() public {
```

Comment:
I've seen this setup a few times, can it be shared in ForkUtils?

Reply:
this might be possible, tbh i havent ever checked!